### ISSUE TITLE
Слетают параметры фильтра в админке при пагинации.

### ISSUE THREAD

**Issue by FAPLAP** [ROSB/Site-Mags#20]
Конкретно было замечено при  использовании фильтрации по Верстальщику. При переходе на следующую страницу Верстальщик слетает.

Выборочно проверил - сайт и даты публикации, эти параметры не слетают.

Гипотеза. Кажется такой глюк (слет фильтра при пагинации) уже был и ты его исправлял! Но это было до ввода понятия Верстальщик и переделки системы Списков. Возможно ты забыл сделать с Верстальщиком то, что делал раньше с другими параметрами Фильтра.

**Comment by SpartakN** [comment]
Сделал. Действительно, забыл в текущий фильтр его добавлять.

**Comment by FAPLAP** [comment]
Что-то у меня ничего не изменилось. Ctrl+f5 жал. Может только dev сделал, а основу не обновил?

**Comment by SpartakN** [comment]
Да. Локально у меня всё работает, но частично обновить сайт не могу. Надо полностью публиковать вместе с изменениями БД и галками на рекламу.

**Comment by SpartakN** [comment]
@FAPLAP 
Проверь, сейчас баг проявляется?
